research-software-directory/RSD-as-a-service
# More fake data

# More fake data with fake releases

Changes proposed in this pull request:

* Generate more fake data, including fake releases
* Software with a real concept DOI will have prefix `Real software` in its name, with no fake releases attached, so they can easily be found and scraped
* The other software, with prefix `Software` in its name, won't have a concept DOI, because otherwhise the scraper would attempt to scrape its releases

How to test:
* `docker-compose down --volumes && docker-compose build --parallel && docker-compose up --scale data-generation=1`.
* Check various pages

PR Checklist:
